integration-tests/scenarios/user_management.md

```diff
@@ -0,0 +1,6 @@
+# Feature: User Management API
+- Scenario: Get UUID via post call
+- Scenario: Trying out the API without user key
+- Scenario: Test PUT call via valid synk token
+- Scenario: Trying with a invalid synk token
+- Scenario: Request for an invalid user
```

Comment:
Can we have scenario of synk token getting expired?

Reply:
yup will add